source/developers/references/developer_guide/process/pull-request-statuses.rst

```diff
@@ -0,0 +1,125 @@
+=========================
+Pull Request Status Guide
+=========================
```

Comment:
@mphilbrick211 @itsjeyd I moved the Pull Request Status Guide from Google docs into the docs site, because it'll be a lot easier to find than a Google doc and more accessible than a PDF file. What do you think? You can see the page here: https://docsopenedxorg--395.org.readthedocs.build/en/395/developers/references/developer_guide/process/pull-request-statuses.html

Reply:
@sarina - looks good to me, except for some formatting (screenshot attached). Not a huge deal, but let me know if it's something you want me to fix.
<img width="826" alt="image" src="https://github.com/openedx/docs.openedx.org/assets/100873803/0ed04ac1-616f-422c-b6ea-111b7e88406c">

Reply:
@sarina Technically we don't have a status called "**In** Product Review" right now; it's just called "Product Review" on the Contributions board. Perhaps that's something we should adjust on the board, though, not in the docs?

CC @mphilbrick211

Reply:
I'll adjust the language to use "Product Review"